proto/cronos/tx.proto

```diff
@@ -3,11 +3,38 @@ package cryptoorgchain.cronos.cronos;
 
 // this line is used by starport scaffolding # proto/tx/import
 
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
 option go_package = "github.com/crypto-org-chain/cronos/x/cronos/types";
 
-// Msg defines the Msg service.
+// Msg defines the Cronos Msg service
 service Msg {
     // this line is used by starport scaffolding # proto/tx/rpc
+
+    // Send defines a method for converting ibc coins to Cronos coins.
+    rpc ConvertTokens(MsgConvertTokens) returns (MsgConvertResponse);
+
+    // Send defines a method to send coins to Crypto.org chain
+    rpc SendToCryptoOrg(MsgSendToCryptoOrg) returns (MsgConvertResponse);
```

Comment:
Is this message only for sending CRO back to crypto-org, or sending any ibc tokens back?

Reply:
both (that is why there is a switch in the logic) 
but we could separate them to different transaction if its too complex

Reply:
I see, maybe change the name to sth like `SendToIBC`, since we are not limited to crypto-org chain?

Reply:
I though all IBC tokens from cronos will comes from crypto-org chain?

If its not the case, design might be even more complex...

Let just focus on the IBC CRO flow in this PR.

I think for the IBC <-> erc20, there are still few thinking that we need to clear

Reply:
User can create ibc channels to whatever chains, I don’t think we can or should enforce that user only interact with crypto.org chain.

Reply:
Yeah but the ERC20 wrapping will be only for the native tokens from crypto-org
This transaction is to do the unwrapping and send back to crypto-org.

For ther IBC use cases, we can define additional transactions.

Anyways as I mention, lets just focus on the CRO token flow in this PR.

Reply:
I changed the name to be more generic